### PR TITLE
Justyn-Clark | Joining the collaboration 

### PR DESCRIPTION
**Pull request by justyn-clark**
Hi Maintainer - Please review my additions to the repo.

### CODE REVIEW

File: justyn-clark.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+			body {padding: 15%; text-align: center; background-color: #e4e4e4; line-height: 1.6em; margin: 0 auto;}
+			p {font-family: sans-serif; font-weight: normal;}
+			a {color:white; -webkit-transition: all .5s ease-in-out; -moz-transition: all .5s ease-in-out; -o-transition: all .5s ease-in-out; transition: all .5s ease-in-out;}
+			a:hover {color: black; -webkit-transition: all .5s ease-in-out; -moz-transition: all .5s ease-in-out; -o-transition: all .5s ease-in-out; transition: all .5s ease-in-out;}
+			h1 {font-size: 6em; color: white; font-family: 'helvetica Neue', sans-serif; font-weight: 100; margin: 0; line-height: 1em;}
+			h2 {font-size: 2em; color: black; font-family: sans-serif; font-weight: normal;}
+			</style>
+	</head>
+	<body>
+		<div>
```
Comment (muan) [inline justyn-clark.html:14]: Looks like this `div` isn't doing much here, can you do without it?

**Comment by muan** [comment]
This looks good to merge! :sparkles: Just made a comment on the `div` but it doesn't really matter, I'll merge later in case you want to change it. :smile:

**Comment by alfredxing** [comment]
:+1:

**Comment by muan** [comment]
OK merging :sparkles: @justyn-clark thanks for the contribution also congrats on your first PR!
See your page here: http://muan.co/hello-world/justyn-clark.html

p.s. feel free to submit another pr if you want to do something about that `div`!

File: justyn-clark.html (hunk 1 above)
Comment (justyn-clark) [inline justyn-clark.html:14]: Very true and I could toss it. Thank you.

**Comment by justyn-clark** [comment]
I'd love to nix that div and submit another pr. Stand by.

**Pull request merged**